test/e2e/prune_test.go

```diff
@@ -13,8 +13,8 @@ import (
 var pruneImage = fmt.Sprintf(`
 FROM  %s
 LABEL RUN podman --version
-RUN apk update
-RUN apk add bash`, ALPINE)
+RUN echo hello > /hello
+RUN echo hello2 > /hello2`, ALPINE)
```

Comment:
This looks like a cut and paste error `, ALPINE)`

Reply:
I think it's OK? That's the argument for the `FROM %s` in the sprintf()?